reactioncommerce/reaction
# [WIP] feat: fix _objectSpread issue

Resolves #5460   
Impact: **critical**  
Type: **bugfix**

## Issue

Cannot open some admin panels because `_objectSpread` is not available. 

## Solution

Update babel/core-js

## Breaking changes

none

## Testing
1. Open http://localhost:3000/operator/tags
2. ensure it runs
